File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> m;
        for(auto &e:nums) m[e]++;
        int n = nums.size();
        vector<vector<int>> cnt(n + 1);
        for(auto &[x, y]:m) cnt[y].push_back(x);
        vector<int> ans;
        for(int i = n; i >= 0; i--){
            for(auto &e:cnt[i]){
                if(k == 0) break;
                ans.push_back(e);
                k--;
            }
            if(k == 0) break;
        }
        return ans;
    }
};

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        ans = []
        n = len(nums)
        for i in range((1 << n)):
            x = []
            for j in range(n):
                if i & (1 << j): x.append(nums[j])
            ans.append(x)
        return ans